Resources/config/services.xml

```diff
@@ -7,6 +7,9 @@
         <parameter key="bazinga_geocoder.geocoder.class">Geocoder\Geocoder</parameter>
         <parameter key="bazinga_geocoder.geocoder.loggable_class">Bazinga\Bundle\GeocoderBundle\Geocoder\LoggableGeocoder</parameter>
         <parameter key="bazinga_geocoder.geocoder.adapter.class">Geocoder\HttpAdapter\CurlHttpAdapter</parameter>
+        <parameter key="bazinga_geocoder.geocoder.adapter.geoip2.class">Geocoder\HttpAdapter\GeoIP2Adapter</parameter>
```

Comment:
Yep, as long as i remember, the bundle was based on geocoder v2, where the adapter was named like this